.github/workflows/bosh.yaml

```diff
@@ -0,0 +1,19 @@
+name: bosh-release
+on:
+  pull_request:
+  workflow_dispatch:  
+jobs:
+  bosh-release:
+    name: Build Autoscaler Components
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Update
+        run: |
+          git submodule foreach --recursive git submodule sync && git submodule update --init --recursive
+
+      - name: Build Dev Release
+        uses: garethjevans/bosh-release-action@v1.0.2
```

Comment:
Should we try to get a repo in the `cloudfoundry` org for this action? Or would you try to get an improved action merged back into orange-cloudfoundry?

Reply:
Am happy to take your guidance on that.  It would be nice to get something official into the `cloudfoundry` org